scripts/libs.js

```diff
@@ -54,7 +59,8 @@ export const metaTags = {
   },
   template: {
     metaName: 'template',
-    // contentType: 'string template name',
+    fallbackContent: '/page-templates/',
```

Comment:
should we make the default just `/templates`?

Reply:
I wanted to be more specific because I want to check if a page is a template just by checking if the path contains /page-templates/. Otherwise finding ways to check this is way more complex. 

And so templates can be at any level but always in 'page-templates' folder

Reply:
I just thought just `/templates/` is already specific enough but as you prefer